examples/ibc/packet_forward_test.go

```diff
@@ -194,26 +195,50 @@ func TestPacketForwardMiddleware(t *testing.T) {
 		require.NoError(t, err)
 		_, err = testutil.PollForAck(ctx, chainA, chainAHeight, chainAHeight+30, transferTx.Packet)
 		require.NoError(t, err)
-		err = testutil.WaitForBlocks(ctx, 1, chainA)
-		require.NoError(t, err)
+		maxBlocks := 5
+		err = testutil.WaitForBlocksUtil(maxBlocks, func(_ int) error {
+			err = testutil.WaitForBlocks(ctx, 1, chainA)
+			require.NoError(t, err)
 
-		chainABalance, err := chainA.GetBalance(ctx, userA.FormattedAddress(), chainA.Config().Denom)
-		require.NoError(t, err)
+			chainABalance, err := chainA.GetBalance(ctx, userA.FormattedAddress(), chainA.Config().Denom)
+			if err != nil {
+				return err
+			}
+			expect := userFunds - transferAmount
+			if expect != chainABalance {
+				return fmt.Errorf("chainA balance mismatch: expect %d, got %d", expect, chainABalance)
+			}
```

Comment:
This is likely not the best example for this as we already have [PollForBalance](https://github.com/mmsqe/interchaintest/blob/wait_util/chain/cosmos/poll.go#L64-L82)

This will likely fit better in miscellaneous test once I get that PR merged in and can revert a check to that